modules/ROOT/nav.adoc

```diff
@@ -31,8 +31,7 @@ include::{partialsdir}/attributes.adoc[]
 ** xref:metrics/index.adoc[{metrics-service}]
 *** xref:metrics/provisioning.adoc[Provisioning]
 *** xref:metrics/binding.adoc[Binding]
-*** xref:configuring-dev-env.adoc[Configuring development environment]
-*** xref:metrics/coding.adoc[Adding Code]
+*** xref:metrics/configuring-dev-env-metrics.adoc[Configuring development environment]
```

Comment:
@finp - any idea why the following will not work (I'm guess it is simply that an xref can't handle this syntax)?

** xref:{partialsdir}/generic-configuring-dev-env.adoc[Configuring development environment]

I tried this too but with no luck: ** xref:_partials/generic-configuring-dev-env.adoc[Configuring development environment]

**Motivation:**

I want to replace two occurrences of '** xref:configuring-dev-env.adoc[Configuring development environment]' in nav.adoc with 'generic-configuring-dev-env.adoc'